subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/result/DefaultResolutionResult.java

```diff
@@ -72,22 +73,7 @@ public void allDependencies(final Closure closure) {
         allDependencies(ConfigureUtil.configureUsing(closure));
     }
 
-    private void eachElement(
```

Comment:
Moved to https://github.com/gradle/gradle/pull/23366/files#diff-222701e87dcd0e3b204770e5ee498f68855ee0f08cac3742e0ccc451bc9b664fR141